content-webapp/src/main/webapp/news-activity-composer-app/main.js

```diff
@@ -30,6 +30,7 @@ const lang = typeof eXo !== 'undefined' ? eXo.env.portal.language : 'en';
 
 // should expose the locale resources as REST API
 const urls = [
+  `${eXo.env.portal.context}/${eXo.env.portal.rest}/i18n/bundle/locale.portlet.notes.notesPortlet-${lang}.json`,
```

Comment:
does this needed for common notes labels ?

Reply:
yes needed to not duplicate the labels of the editor
for the translation components for example